File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyTableViewElementsRow.cpp

#include "SelectRowCommand_ViewMyTableViewElementsRow.h"
#include "SelectRowCommand_ViewMyTableViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewMyTableViewElementsRow::getColumnCellTableColumnText( ) 
  {
    return this->columnCellTableColumnText;
  }
  std::string SelectRowCommand_ViewMyTableViewElementsRow::getColumnCellLabelText( ) 
  {
    return this->columnCellLabelText;
  }
  int SelectRowCommand_ViewMyTableViewElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyTableViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewMyTableViewElementsRow.cs
/// </filename>


namespace commands
{
    internal class SelectRowCommand_ViewMyTableViewElementsRow
  {
        private string columnCellLabelText;
     
        public     virtual string getColumnCellLabelText()
    {
      return this.columnCellLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

